.vscode/tasks.json

```diff
@@ -4,14 +4,16 @@
     "version": "2.0.0",
     "tasks": [
         {
-            
-            "label": "compile debugopt",
+            "label": "compile debug",
             "type": "shell",
-            "command": "cmake --build .",
```

Comment:
I am not using vscode, so I don't really care. but this will fail when you run it on an empty repo. because cmake by default generates Makefiles, but then you use ninja and not make....

if you want to use ninja, you need `-GNinja` added to cmake. and anyway `cmake --build` is better than `ninja` because it automatically picks the build system you have chosen.

also debug build is really slow. maybe debugopt?